src/assets/css/Article.scss

```diff
@@ -0,0 +1,91 @@
+
```

Comment:
please, use indentation of 2 spaces in all your files

Reply:
on it

Reply:
Resolved